lib/linguist/heuristics.yml

```
@@ -88,6 +88,16 @@ disambiguations:
     pattern: '(<^\s*; |End Function)'
   - language: BitBake
     pattern: '^\s*(# |include|require)\b'
+- extensions: ['.bf']
+  rules:
+  - language: Beef
+    pattern: '^using\s+(System|Beefy)(\.(.*))?;\s*$'
```

For performance's sake, I recommend disabling multiline mode so `(.*)` won't scan any further than 1 line:

```suggestion
    pattern: '(?-m)^\s*using\s+(System|Beefy)(\.(.*))?;\s*$'
```